src/Database/Console/Migrations/FreshCommand.php

```diff
@@ -75,4 +75,31 @@ protected function getOptions()
             $this->addWebsiteOption()
         ]);
     }
+
+    protected function dropAllTables($database)
+    {
+        if ($this->parentMethodExists($this, "dropAllTables")) {
+            parent::dropAllTables($database);
+
+            return;
+        }
+
+        $this->call('db:wipe', array_filter([
+            '--database' => $database,
+            '--drop-views' => $this->option('drop-views'),
+            '--drop-types' => $this->option('drop-types'),
+            '--force' => true,
+        ]));
+    }
+
+    protected function parentMethodExists($object, $method)
+    {
+        foreach (class_parents($object) as $parent) {
+            if (method_exists($parent, $method)) {
+                return true;
+            }
+        }
+
+        return false;
+    }
```

Comment:
I added this work-around, because the `dropAllTables()` method no longer exists in Laravel 6. Feel free to optimize. :)

Reply:
Looks about how I'd do it, too. 👍 

Ref to Laravel 6 commit: https://github.com/laravel/framework/commit/c26b435d826ff7882a0185eb5e48001683d48b9d

Reply:
Don't add backward compatibility logic. Just remove that and use db:wipe. Potentially remove fresh command completely and implement a tenancy:db:wipe one?